coursier.rb

```diff
@@ -2,9 +2,15 @@
 class Coursier < Formula
   desc "Launcher for Coursier"
   homepage "https://get-coursier.io"
-  url "https://github.com/coursier/coursier/releases/download/v2.1.7/cs-x86_64-apple-darwin.gz"
   version "2.1.7"
-  sha256 "db49faf31b5ef824394530c4ba6b1d584604e98a546502b15b8a6f0488e78d26"
+  on_intel do
+    url "https:a//github.com/coursier/coursier/releases/download/v2.1.7/cs-x86_64-apple-darwin.gz"
+    sha256 "db49faf31b5ef824394530c4ba6b1d584604e98a546502b15b8a6f0488e78d26"
+  end
+  on_arm do
+    url "https://github.com/VirtusLab/coursier-m1/releases/download/v2.1.7/cs-aarch64-apple-darwin.gz"
```

Comment:
```suggestion
    url "https://github.com/coursier/coursier/releases/download/v2.1.7/cs-aarch64-apple-darwin.gz"
```

Coursier recently enabled the arm workflow on the main repo